Dockerfile

```
@@ -0,0 +1,19 @@
+FROM ubuntu:21.04
+
+RUN apt update && apt install -y ca-certificates
+
+workdir /basilisk
```

```suggestion
WORKDIR /basilisk
```


note that you'll have to change following command accordingly (remove basilisk from path)